### ISSUE TITLE
List View for tui.calendar

### ISSUE THREAD

**Issue by Aurugorn** [nhnent/tui.calendar#213]
Is there a possibility to change view to List View?
Something like this:
![listview](https://user-images.githubusercontent.com/20743612/46070899-21fc7280-c17f-11e8-9efe-a1e1c50aea6e.png)

If not, anyone know how to create this feature? Would love it.

**Comment by dongsik-yoo** [comment]
Oh, thank you for your attention. Some guys wanted this view type and we've been discuss it. But we don't have any plan to implement a list view yet.

**Comment by hankshrader** [comment]
Hi, is there any chance it will be implemented soon?

**Comment by netzperfekt** [comment]
would love this, too!

**Comment by adhrinae** [comment]
It won't be implemented soon. Sorry for that.

**Comment by stanverbakel** [comment]
Yes this feature would be really nice for mobile, the other views don't quite fit into the screen
